### ISSUE TITLE
class-weighted loss for imbalance

### ISSUE THREAD

**Issue by tagny** [lengocluyen/relation_extraction_textmine25#10]
- https://stackoverflow.com/questions/70979844/using-weights-with-transformers-huggingface
- https://discuss.huggingface.co/t/how-can-i-use-class-weights-when-training/1067/7
- https://github.com/huggingface/transformers/issues/9374
- https://discuss.pytorch.org/t/weight-vs-pos-weight-in-nn-bcewithlogitsloss/114859#post_2
